src/hooks/useBridgesApi/useGetBridgesApi.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  BridgesApi,
+  Configuration,
+  BridgeListResponse,
+} from "@openapi/generated";
+import { useState } from "react";
+
+export function useGetBridgesApi(
+  getToken: () => Promise<string>,
+  basePath: string
+): {
+  getBridges: (pageReq?: number, sizeReq?: number) => Promise<void>;
+  bridges?: BridgeListResponse[];
+  page?: number;
+  size?: number;
+  total?: number;
+  isLoading: boolean;
+  error: unknown;
+} {
+  const [page, setPage] = useState<number>();
+  const [size, setSize] = useState<number>();
+  const [total, setTotal] = useState<number>();
+  const [bridges, setBridges] = useState<BridgeListResponse[]>();
```

Comment:
I think my previous comment about this response type was wrong. 
You are actually setting only the `items` here from the `getBridges` response. So it should be `BridgeResponse[]`. Right?
Sorry about it :sweat: